### PR TITLE
Display patient age instead of hard coded '58' Fixes #104

### LINKED ISSUE #104: [Dashboard] Patient age on linked provider detail page
Click on Dashboard. At the bottom, locate your provider. Click the '>' at the far right end of the provider entry. Review your patient data at the top of the next screen.

The birthday shown for me is correct, but the age is off by several years (also, the unit shown is "year," not "years"). I checked the JSON for the patient details and didn't see my age hardcoded in the JSON.

### PR DESCRIPTION
**Pull request by Serinus1**

### CODE REVIEW

**Comment by Serinus1** [comment]
https://jsfiddle.net/dj2y1tzb/

**Comment by AnalogJ** [comment]
Thanks for the PR 🥳 

A large chunk of the current dashboard will actually be removed, and replaced with user configurable widgets - see https://github.com/fastenhealth/fasten-onprem/issues/21

However that work is still in early stages, so there's no downside in making this fix for now

File: frontend/src/app/pages/source-detail/source-detail.component.ts [hunk 1]
```diff
@@ -64,17 +64,18 @@ export class SourceDetailComponent implements OnInit {
   }
   getPatientDOB(){
     return getPath(this.selectedPatient?.resource_raw, 'birthDate')
-
+  }
+  getPatientAge(){
+    var msInYear = 365 * 24 * 60 * 60 * 1000;
+    var age = Date.now() - getPatientDOB();
+    return Math.floor(age / msInYear);
```
Comment (AnalogJ) [inline frontend/src/app/pages/source-detail/source-detail.component.ts:71]: This function should gracefully handle invalid/null values in `getPatientDOB()`

getPatientDOB may be null, a String or a Date.

**Review by AnalogJ — COMMENTED** [review]
some error handling required.

**Comment by Serinus1** [comment]
Apologies.  Fixed and did some testing in an updated [jsfiddle](https://jsfiddle.net/dj2y1tzb/4/).  Also handles strings that look like dates.

**Comment by AnalogJ** [comment]
Sorry about the delay merging this PR, I was a bit busy with the `sandbox` branch.

**Pull request merged**